docs/source/model_design/vivarium_features/cause_of_death_selection/index.rst

```diff
@@ -0,0 +1,216 @@
+--------------------------
+
+Instead of the above, we want to transfer probability *directly* from the intervened-on
+cause to "does not die."
+In order to do this, we reserve sections of the probability space for each
+potential cause of death in the model.
+**These reservations are the same between scenarios.**
+The probability section for cause B never starts until after the *reserved* section
+for cause A, regardless of the scenario-specific probability of cause A.
+
+.. image:: cod_solution_diagram.drawio.svg
+
+Making the reservations the same between scenarios is challenging in Vivarium,
+due to the fact that scenarios run independently in an embarrassingly parallel fashion,
+with no inter-scenario communication.
+As such, we can only make the reservations based on information that we guarantee
+we will not change between scenarios.
+We typically change risk factor exposures and cause states, and sometimes directly
+decrease excess mortality rate for a given cause state and risk factor exposure,
+in our intervention scenarios.
+However, we can generally assume that we will not add new causes, add new risk factor exposure
+levels or cause states, change demographics such as age, or directly *increase* mortality rates
+(conditional on risk exposures and cause states) in our intervention scenarios.
+
+Therefore, for each age group and sex of simulants,
+we can reserve for each cause its maximum cause-specific mortality probability (for that age group and sex) across
+all possible risk factor exposures and cause states, and before applying any interventions on the mortality
+probability directly.
+This will be consistent between scenarios because age group and sex will not change, and the *possible*
+risk factor exposures and cause states will not change (though our intervention may change their prevalences).
+For continuous risk factors, we will need to bound the highest-risk exposure to some plausible range
+for these purposes.
+These are conservative upper-bound reservations that could be tightened if there was more
+inter-scenario information sharing.
+
+If our reservations under this scheme would require us to reserve more than 100% probability,
+we will scale down our reservations proportionally to fit under 100%.
+This means that our reservations may not be big enough all of the time and it is theoretically
+possible for the true probability to "overflow" the reservation.
+In this case, we revert to the status quo: an overflowing probability will shift over the next
+cause's probability.
+As long as this happens rarely, we will still get good variance reduction in aggregate.
+If this is common, it may indicate that our timestep is too long.
+
+.. note::
+
+  I believe we could get slightly better variance reduction by starting each cause's probability
+  at the midpoint of its reservation, and growing symmetrically outward toward the edges of the
+  reservation, then overflowing the reservation in a random direction (e.g. right).
```

Comment:
Do you mean "random" or "arbitrary"? I.e., would we randomly pick the direction each time, or do we arbitrarily pick either left or right, and always use that?

Also, why exactly would this lead to more variance reduction?

Reply:
Best would be random, at the cause level, I think.

I don't want to go too far down this rabbit hole, but I got to this suggestion by thinking about how we could _minimize_ the probability that one cause's probability allocation "bumps into" (and therefore shifts) another. Starting everything at the left side of its reservation, as I've actually proposed here, means that anytime a cause overflows and the next is non-zero, shifting occurs. The midpoint expansion approach means that a shift would require overflow > (next_cause_reservation - next_cause_probability) / 2. But it's possible I've totally missed something